docs/book/src/reference/cli-plugins.md

```diff
@@ -0,0 +1,262 @@
+# CLI Plugins
```

Comment:
missing add the menu. I am unable to find it here: https://deploy-preview-1539--kubebuilder.netlify.app/